x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -324,14 +326,19 @@ func (m *MetricSet) readCloudwatchConfig() (listMetricWithDetail, map[string][]n
 			!configDimensionValueContainsWildcard(config.Dimensions) {
 			namespace := config.Namespace
 			for i := range config.MetricName {
-				metricsWithStats := metricsWithStatistics{
-					cloudwatchMetric: types.Metric{
+				cwMetric := aws.MetricWithID{
```

Comment:
what's the reason we need to use this type here instead of a plain old `Metric`?

Reply:
Good catch! It should be a plain old `Metric`!